src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -31,6 +30,8 @@ export class CourseExerciseDetailsComponent implements OnInit {
     public sortedResults: Result[];
     public sortedHistoryResult: Result[];
 
+    getIcon = getIcon;
```

Comment:
is this necessary? It looks weird and wrong

Reply:
I agree, but unfortunately there isn't any other way, since the template can use only what the component exposes

Reply:
I have the same understanding, unfortunately.